### ISSUE TITLE
PhyloP and Phast Cons widgets show no scores (and other complains)

### ISSUE THREAD

**Issue by diekhans** [KarchinLab/open-cravat#164]
```
Genome: hg38
Example: TSV
Annotations:
  Evolution:
     PhastCons
     PhyloP
```
```

submit job, when done go to Variant

Layout: hide all
        show: PhastCons, PhyloP, Rank Score

Select ENST0000371958

Bug:
  PhyloP and Phast Cons widgets show no scores,
  whoever the Rank Score widget has phystcons and phylop
  scores that are in red.

Also, there I have no idea what the red means.  Having
a help icon on widgets would be really useful.

Also, there the red background is too dark and not enough contrast
with the numbers, make them hard to read.

```

**Comment by diekhans** [comment]
<img width="1342" alt="Screenshot 2023-07-13 at 8 13 06 PM" src="https://github.com/KarchinLab/open-cravat/assets/110053/c78f15f2-6c51-4f46-ba8f-6c708818fed9">

**Comment by kmoad** [comment]
This looks like an issue with the Rank Score widget. It's having issues updating when a new row is selected. Will update here when it's fixed.

**Comment by mlarsen5** [comment]
This issue has been fixed in Rank Score widget version 1.1.0, now always reporting the correct score. The color is a gradient scale ranging from white to red, the higher the score the more red the cell is. We also addressed the issue that there is not enough contrast between the background and the number when the color is too dark.
